### ISSUE TITLE
How does the learning rate have anything to do with the bias?

### ISSUE THREAD

**Issue by ghost** [kochbj/Deep-Learning-for-Causal-Inference#2]
In [Tutorial 1](https://colab.research.google.com/drive/1Zx0AkriygB_ws6qXjA7VfqebG-YMwbWl#scrollTo=ixInwwcKmMfO), there is the following saying:

> While you should experiment with different learning rates, I recommend having a conservative (smaller) learning rate because we really want our estimator to be unbiased.

Here, "unbiased" means that *no selection bias* or that the estimator converges around a global minimum?

**Comment by kochbj** [comment]
Hi Ywandung,

Definitely not selection bias. I guess this would literally be bias from a statistical learning perspective (i.e., error in-sample). If your learning rate is too high, you risk overshooting optima and ending up with a poorly fitting model. Learning rate schedulers help, but it's better to be conservative.

I think this is probably the largest weakness of the models and why there aren't any empirical applications that I've seen: theoretical guarantees (whether they be generalization bounds, statistical bias or consistency ) are mitigated if your model ends up at different places depending on the training run and hyperparameter settings. 

Hope this helps.

-B
